cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -143,6 +145,17 @@ open class Node @Inject constructor(private val project: Project) {
         @Optional @Input get
         private set
 
+    // Should the cordform set up run data base migration scripts after installation - defaults to false for compatibility
+    // with current and previous Corda versions
+    @get:Optional
+    @get:Input
+    var runSchemaMigration: Boolean = false
```

Comment:
Modern Gradle properties are supposed to be "lazy" properties whose values are computed when Gradle's configuration phase is complete.
```kotlin
@get:Optional
@get:Input
val runSchemaMigration: Property<Boolean> = project.objects.property(Boolean::class.javaObjectType).convention(false)
```
or something like that.

Reply:
I've done that change - I don't like the required changes to the build.gradle file though - I am not sure that the realised benefits are worth changing
```
runSchemaMigration = true
```
to 
```
runSchemaMigration.set(true)
```